tests/foreman/api/test_location.py

```diff
@@ -247,23 +249,21 @@ def test_positive_create_update_and_remove_capsule(self):
 
         :CaseImportance: Critical
         """
-        proxy_id_1 = self._make_proxy()['id']
-        proxy_id_2 = self._make_proxy()['id']
+        proxy_id_1 = make_proxies['proxy1']['id']
+        proxy_id_2 = make_proxies['proxy2']['id']
```

Comment:
This could be replaced with list comprehension and unpacking:
```python
proxy_id_1, proxy_id_2 = [proxy['id'] for proxy in make_proxies.values()]
```